static/app/bootstrap/initializeApp.tsx

```diff
@@ -1,6 +1,4 @@
-import 'bootstrap/js/alert';
-import 'bootstrap/js/tab';
```

Comment:
https://github.com/twbs/bootstrap/blob/v3.4.1/js/tab.js

Reply:
I re-implemented the functionality we use, which is basically:

1. React to click events that target `[data-toggle="tab"]` elements.
2. When they are clicked find the target toggle element by looking at the `href` and use this as the ID to lookup in the document.
3. Remove the current active panel and active tab classes (and `aria-expanded` attributes).
4. Set the new active panel as `.active` and mark the tab anchor with `aria-expanded`